src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```
@@ -75,7 +75,7 @@
       This prevents an early error message during 'dotnet publish'.
       We handle $(SelfContained) in a custom way where it is forced to be true.
     -->
-    <PublishSelfContained Condition=" '$(PublishSelfContained)' == '' ">false</PublishSelfContained>
+    <PublishSelfContained Condition=" '$(PublishSelfContained)' == '' and $([MSBuild]::VersionLessThanOrEquals($(NETCoreSdkVersion), '9.0')) ">false</PublishSelfContained>
```

Wait let me retest this, seems like it should be:
```suggestion
    <PublishSelfContained Condition=" '$(PublishSelfContained)' == '' and $([MSBuild]::VersionLessThan($(NETCoreSdkVersion), '9.0')) ">false</PublishSelfContained>
```